src/modules/groups/groups.js

```diff
@@ -114,7 +76,7 @@ router.get('/all/:type', async (req, res) => {
       data: { value }
     } = await axios.get(URL, {
       headers: {
-        Authorization: `Bearer ${access_token}`
+        Authorization: `Bearer ${accessToken}`
```

Comment:
good to keep Bearer concatenation in getaccesstoken method

Reply:
M not following this. Can you rephrase ?

Reply:
abstract concatenating Bearer string in getAccessToken function

Reply:
its can return `Bearer undefined`